tools/proto/buf.yaml

```diff
@@ -1,4 +1,6 @@
 version: v1beta1
+deps:
+  - "buf.build/k8s/api"
```

Comment:
```suggestion
  - "buf.build/k8s/api"
```

Does this create a dependency on the `buf` registry being up/available? I believe it does.

Is it possible to vendor this deps, as we do with other things?

Reply:
we already have it vendored in common-protos (or at least the ability to, probably only the API repo uses it today). +1 to vendoring

Reply:
Thank you @bleggett and @howardjohn for your feedback. I'm going to go on that direction of vendoring this dep.